<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="REVATHIBABU" failures="2" tests="2" name="Test" time="1.281" errors="0" timestamp="24 Mar 2020 15:06:50 GMT">
  <testcase classname="vsc.test.LoginTest" name="Login" time="0.025">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at vsc.objectRepository.PageProvider.launch(PageProvider.java:34)
	at vsc.test.LoginTest.Login(LoginTest.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="vsc.test.LoginTest" name="Login" time="0.006">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at vsc.objectRepository.PageProvider.launch(PageProvider.java:34)
	at vsc.test.LoginTest.Login(LoginTest.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
</testsuite> <!-- Test -->
